spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -237,29 +233,17 @@ class SpotSemanticPlaceEEEnv(SpotSemanticPlaceEnv):
     """This is Spot semantic place class"""
 
     def __init__(self, config, spot: Spot, use_semantic_place: bool = False):
-        # We set the initial arm joints
-        config.INITIAL_ARM_JOINT_ANGLES = copy.deepcopy(
-            config.INITIAL_ARM_JOINT_ANGLES_SEMANTIC_PLACE
-        )
+
         super().__init__(config, spot)
 
         # Define End Effector Policy Scale Values
         self.arm_ee_dist_scale = self.config.EE_DIST_SCALE_SEMANTIC_PLACE
         self.arm_ee_rot_scale = self.config.EE_ROT_SCALE_SEMANTIC_PLACE
 
-        # Place steps
-        self._time_step = 0
-
-    def get_success(self, observations):
-        return self.place_attempted
-
     def get_observations(self):
         assert self.initial_ee_pose is not None
         assert self.target_object_pose is not None
 
-        # Get the gaol sensor
-        obj_goal_sensor = self.get_place_sensor(True)
-
         # Get the delta ee orientation
         current_gripper_orientation = self.spot.get_ee_quaternion_in_body_frame()
```

Comment:
we don't need all the other stuff in get_observations anymore, just this part:
```
xyz, rpy = self.spot.get_ee_pos_in_body_frame()
        observations = super().get_observations()
        if "joint" in observations:
            del observations["joint"]
        xyz, rpy = self.spot.get_ee_pos_in_body_frame()
        observations["ee_pose"] = np.concatenate([xyz, rpy])
```
since `observations = super().get_observations()` will get us the observations from SemanticPlaceEnv, which gets all the other sensor data